### PR TITLE
Translate doc certificate-rotation into Chinese

### PR DESCRIPTION
**Pull request by lichuqiang**
CN doc translation

/cc @tengqm @zhangxiaoyu-zidif

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6343)

### CODE REVIEW

**Comment by zhangxiaoyu-zidif** [comment]
wait for @tengqm confirmation

File: cn/docs/tasks/tls/certificate-rotation.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+当 kubelet 启动时，如被配置为自举（使用`--bootstrap-kubeconfig` 参数），kubelet 会使用其初始证书连接到
+Kubernetes API ，并发送证书签名的请求。 可以通过以下方式查看证书签名请求的状态：
+
+```sh
+kubectl get csr
+```
+
+最初，来自节点上 kubelet 的证书签名请求处于 `Pending` 状态。 如果证书签名请求满足特定条件，
+控制管理器会自动批准，此时请求会处于 `Approved` 状态。 接下来，控制器管理器会签署证书，
```
Comment (tengqm) [inline cn/docs/tasks/tls/certificate-rotation.md]: 前面一个
控制管理器->控制器管理器

File: cn/docs/tasks/tls/certificate-rotation.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+证书的有效期限由 `--experimental-cluster-signing-duration` 参数指定，签署的证书会被附加到证书签名请求中。
+
+Kubelet 会从 Kubernetes API 取回签署的证书，并将其写入磁盘，存储位置通过 `--cert-dir` 参数指定。
+然后 kubelet 会使用新的证书连接到 Kubernetes API。
+
+当签署的证书即将到期时，kubelet 会使用 Kubernetes API，发起新的证书签名请求。
+同样地，控制管理器会自动批准证书请求，并将签署的证书附加到证书签名请求中。 Kubelet
```
Comment (tengqm) [inline cn/docs/tasks/tls/certificate-rotation.md]: 控制器管理器

**Comment by lichuqiang** [comment]
@tengqm @zhangxiaoyu-zidif done :)

**Pull request merged**